pages/common/az-storage-blob.md

```diff
@@ -0,0 +1,25 @@
+# az storage blob
+
+> Manage blob storage containers and objects in Azure.
+> Part of `azure-cli`.
+> More information: <https://docs.microsoft.com/cli/azure/storage/blob>.
+
+- Download a blob to a file path:
+
+`az storage blob download --account-name {{storage_account_name}} --account-key {{storage_account_key}} -c {{container_name}} -n {{path/to/blob}} -f {{./local_name}}`
+
+- Download blobs from a blob container recursively:
+
+`az storage blob download-batch --account-name {{storage_account_name}} --account-key {{storage_account_key}} -s {{container_name}} -d {{ path/to/remote}} --pattern {{filename_regex}}`
+
+- Upload a local file to blob storage:
+
+`az storage blob upload --account-name {{storage_account_name}} --account-key {{storage_account_key}} -c {{container_name}} -n {{path/to/blob}} -f {{./local_name}}`
+
+- Delete a blob object:
+
+`az storage blob delete --account-name {{storage_account_name}} --account-key {{storage_account_key}} -c {{container_name}} -n {{path/to/blob}}`
+
+- Generate a shared access signature for a blob:
+
+`az storage blob generate-sas --account-name {{storage_account_name}} --account-key {{storage_account_key}} -c {{container_name}} -n {{path/to/blob}} --permissions r --expiry {{Y-m-d'T'H:M'Z'}} --https-only`
```

Comment:
Is `r` a typo?
```suggestion
`az storage blob generate-sas --account-name {{storage_account_name}} --account-key {{storage_account_key}} -c {{container_name}} -n {{path/to/blob}} --permissions --expiry {{Y-m-d'T'H:M'Z'}} --https-only`
```

Reply:
r is the permission literal for read. I changed it a param.

Reply:
You can also use something like `{{r}}` to represent a commonly used permission set.